src/index.ts

```diff
@@ -50,6 +50,8 @@ async function run() {
         report: core.getInput('head-bundle-analysis-report-path'),
       },
       githubToken: core.getInput('github-token'),
+      targetedBundles: core.getInput('targeted-bundles').split(',').map((bundleItem) => ({ name: `${bundleItem.split(':')[0]}.js`, budget: bundleItem.split(':')[1]})),
```

Comment:
**suggestion**: Managing [the budget list in Gonfalon](https://github.com/launchdarkly/gonfalon/pull/18286/files#diff-37d2bc9e1175c0384da41bae964d8c14922cd18a1e62b92abe1d73c2cb0bfe5aR124) as a string will quickly become cumbersome, assuming we adopt this proposal.

Could we use a YAML list of dictionaries instead?

Reply:
please see my comment on the P.R above, I addressed the constraints there (wish I could use lists)